TypeCobol.Test/Utils/Comparator.cs

```diff
@@ -533,6 +533,15 @@ public override bool Visit(TruncateStatement truncateStatement)
 
                 return true;
             }
+
+            public override bool Visit(WhenEverStatement whenEverStatement)
+            {
+                _writer.WriteLine($"line {whenEverStatement.Line}: {nameof(WhenEverStatement)}");
+                DumpObject(nameof(whenEverStatement.ExceptionCondition), whenEverStatement.ExceptionCondition);
+                DumpObject(nameof(whenEverStatement.BranchingType), whenEverStatement.BranchingType);
+                DumpObject(nameof(whenEverStatement.TargetSectionOrParagraph), whenEverStatement.TargetSectionOrParagraph);
```

Comment:
Now that i read it, could you merge the 2 methods `DumpObject`  and `DumpSqlObject` into a single one ?